store/postgres/src/entities.rs

```diff
@@ -20,16 +20,16 @@
 // for dynamic tables.
 
 use diesel::connection::SimpleConnection;
+use diesel::debug_query;
 use diesel::deserialize::QueryableByName;
 use diesel::dsl::{any, sql};
 use diesel::pg::{Pg, PgConnection};
-use diesel::sql_types::{Bool, Integer, Jsonb, Nullable, Text};
+use diesel::sql_types::{Integer, Jsonb, Nullable, Text};
 use diesel::BoolExpressionMethods;
 use diesel::Connection as _;
 use diesel::ExpressionMethods;
-use diesel::{debug_query, insert_into};
 use diesel::{OptionalExtension, QueryDsl, RunQueryDsl};
-use diesel_dynamic_schema::{schema, Column, Table as DynamicTable};
+use diesel_dynamic_schema::{schema as dds_schema, Column, Table as DynamicTable};
```

Comment:
This PR uses `dds` to mean both "dynamic data source" and "diesel dynamic schema", this name is only used once, I don't mind not importing and writing `diesel_dynamic_schema::schema`.

Reply:
Good point. Changing that.